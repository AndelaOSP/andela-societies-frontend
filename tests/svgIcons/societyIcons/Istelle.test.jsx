import React from 'react';
import { shallow } from 'enzyme';

import IstelleIcon from '../../../src/components/svgIcons/societyIcons/Istelle';

describe('<IstelleIcon />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<IstelleIcon />);
    expect(wrapper).toMatchSnapshot();
  });
});
