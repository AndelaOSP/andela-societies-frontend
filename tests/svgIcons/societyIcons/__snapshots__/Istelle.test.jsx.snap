// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IstelleIcon /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <defs>
          <style>
            .cls-1{fill:#fff;}
          </style>
        </defs>,
        <title>
          istelle-white-icon
        </title>,
        <path
          className="cls-1"
          d="M239,317.54c4.92,0.32,9.84.63,14.76,1q8.71,0.62,17.42,1.3c3.31,0.26,6.63.47,9.93,0.88,5.52,0.68,9,3.85,10.66,9.11a12.42,12.42,0,0,1-3.62,14.09q-11.6,10-23.24,19.92c-2.63,2.25-5.3,4.45-8,6.67l-0.71.57c-0.37-1.37-.76-2.64-1.05-3.93a17.53,17.53,0,0,1-.21-7.82,7.82,7.82,0,0,1,2.53-4.31l23.24-20.12c0.79-.68,1.57-1.38,2.52-2.22-0.88-.1-1.57-0.18-2.26-0.24-4.42-.4-8.84-0.8-13.26-1.18q-6-.51-12-1c-2.11-.16-4.23-0.26-6.34-0.42A8.2,8.2,0,0,1,243.9,327a22.4,22.4,0,0,1-4.62-8.52c-0.07-.22-0.19-0.44-0.28-0.65v-0.27Z"
          transform="translate(-239 -257.91)"
        />,
        <path
          className="cls-1"
          d="M278.66,387.89c1.88-4.77,3.73-9.55,5.65-14.3q4.93-12.21,9.93-24.4a12.23,12.23,0,0,1,6.07-6.77,14.17,14.17,0,0,1,7-1.19,12.53,12.53,0,0,1,6.32,1.92,12.08,12.08,0,0,1,4.59,5.62q4.89,11.83,9.67,23.7c2.07,5.12,4.05,10.28,6.07,15.42h-2.55c-1.52-.23-3.05-0.37-4.54-0.72-3.76-.87-6.53-2.74-8.05-6.62-3.93-10-8.14-19.87-12.25-29.79a5.1,5.1,0,0,0-.28-0.49c-0.09.17-.15,0.27-0.19,0.37q-6.38,15.48-12.75,31a7.84,7.84,0,0,1-2.55,3.37,13.48,13.48,0,0,1-5.63,2.36c-1.15.23-2.32,0.38-3.48,0.56h-3.08Z"
          transform="translate(-239 -257.91)"
        />,
        <path
          className="cls-1"
          d="M291.26,257.91c1,4,1.94,7.92,2.87,11.89,2.25,9.66,4.51,19.31,6.7,29,1.28,5.67-.77,10.42-5.79,13.85a11.89,11.89,0,0,1-13.46.62c-4.3-2.63-8.62-5.24-12.88-7.92-7.25-4.57-14.47-9.19-21.7-13.79l-0.51-.35c0.52-.39,1-0.76,1.47-1.1a20.09,20.09,0,0,1,8.16-3.67,8,8,0,0,1,5.73,1q11.37,7.07,22.75,14.11c1.71,1.06,3.43,2.09,5.15,3.13,0.17,0.1.34,0.18,0.65,0.34-0.22-1-.4-1.91-0.6-2.79-2.18-9.65-4.2-19.34-6.63-28.93-1.22-4.8.48-8.32,3.59-11.54,1.22-1.26,2.65-2.32,4-3.47,0.13-.11.25-0.24,0.37-0.36h0.13Z"
          transform="translate(-239 -257.91)"
        />,
        <path
          className="cls-1"
          d="M373,317.27c-0.28.77-.55,1.54-0.83,2.31a21.18,21.18,0,0,1-3.68,6.56,8.7,8.7,0,0,1-6.71,3.2c-2.87.07-5.74,0.38-8.61,0.6s-5.78.46-8.67,0.71l-8.46.75-5.13.45-1.39.15c1.16,1,2.2,2,3.24,2.9l20.64,18.2a5.4,5.4,0,0,0,.51.43c2.73,1.86,3.8,4.58,3.91,7.75a25,25,0,0,1-1.51,8.92,1.54,1.54,0,0,1-.16.26c-1.52-1.29-3-2.55-4.5-3.83Q338.23,355,324.83,343.29a11.74,11.74,0,0,1-4.43-9.7c0.22-4.15,1.54-7.92,4.79-10.75a11.7,11.7,0,0,1,6.78-2.73q5.09-.49,10.19-0.9c3.27-.26,6.54-0.48,9.81-0.71,3.63-.26,7.25-0.52,10.88-0.76,3.38-.22,6.77-0.41,10.15-0.61v0.13Z"
          transform="translate(-239 -257.91)"
        />,
        <path
          className="cls-1"
          d="M319.17,304.74c1-.56,1.84-1,2.69-1.53l24.94-14.47a12.09,12.09,0,0,1,3.89-1.88,10.62,10.62,0,0,1,5,.35,24.63,24.63,0,0,1,8.25,4.58c-0.17.13-.29,0.24-0.43,0.32l-35,20.69a13.28,13.28,0,0,1-6.51,2,12.27,12.27,0,0,1-7.49-2.45,13.3,13.3,0,0,1-5.71-7.58,13.17,13.17,0,0,1,.1-6.68q5.12-19.65,10.28-39.28c0-.17.11-0.33,0.21-0.65,1.12,0.94,2.21,1.78,3.22,2.71a17.09,17.09,0,0,1,4.33,5.81,8.57,8.57,0,0,1,.37,5.74c-2.11,8.05-4.14,16.13-6.19,24.2-0.62,2.46-1.21,4.93-1.81,7.39C319.26,304.19,319.24,304.39,319.17,304.74Z"
          transform="translate(-239 -257.91)"
        />,
      ],
      "data-name": "Layer 1",
      "id": "Layer_1",
      "viewBox": "0 0 134 129.98",
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <style>
            .cls-1{fill:#fff;}
          </style>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": ".cls-1{fill:#fff;}",
          },
          "ref": null,
          "rendered": ".cls-1{fill:#fff;}",
          "type": "style",
        },
        "type": "defs",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "istelle-white-icon",
        },
        "ref": null,
        "rendered": "istelle-white-icon",
        "type": "title",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M239,317.54c4.92,0.32,9.84.63,14.76,1q8.71,0.62,17.42,1.3c3.31,0.26,6.63.47,9.93,0.88,5.52,0.68,9,3.85,10.66,9.11a12.42,12.42,0,0,1-3.62,14.09q-11.6,10-23.24,19.92c-2.63,2.25-5.3,4.45-8,6.67l-0.71.57c-0.37-1.37-.76-2.64-1.05-3.93a17.53,17.53,0,0,1-.21-7.82,7.82,7.82,0,0,1,2.53-4.31l23.24-20.12c0.79-.68,1.57-1.38,2.52-2.22-0.88-.1-1.57-0.18-2.26-0.24-4.42-.4-8.84-0.8-13.26-1.18q-6-.51-12-1c-2.11-.16-4.23-0.26-6.34-0.42A8.2,8.2,0,0,1,243.9,327a22.4,22.4,0,0,1-4.62-8.52c-0.07-.22-0.19-0.44-0.28-0.65v-0.27Z",
          "transform": "translate(-239 -257.91)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M278.66,387.89c1.88-4.77,3.73-9.55,5.65-14.3q4.93-12.21,9.93-24.4a12.23,12.23,0,0,1,6.07-6.77,14.17,14.17,0,0,1,7-1.19,12.53,12.53,0,0,1,6.32,1.92,12.08,12.08,0,0,1,4.59,5.62q4.89,11.83,9.67,23.7c2.07,5.12,4.05,10.28,6.07,15.42h-2.55c-1.52-.23-3.05-0.37-4.54-0.72-3.76-.87-6.53-2.74-8.05-6.62-3.93-10-8.14-19.87-12.25-29.79a5.1,5.1,0,0,0-.28-0.49c-0.09.17-.15,0.27-0.19,0.37q-6.38,15.48-12.75,31a7.84,7.84,0,0,1-2.55,3.37,13.48,13.48,0,0,1-5.63,2.36c-1.15.23-2.32,0.38-3.48,0.56h-3.08Z",
          "transform": "translate(-239 -257.91)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M291.26,257.91c1,4,1.94,7.92,2.87,11.89,2.25,9.66,4.51,19.31,6.7,29,1.28,5.67-.77,10.42-5.79,13.85a11.89,11.89,0,0,1-13.46.62c-4.3-2.63-8.62-5.24-12.88-7.92-7.25-4.57-14.47-9.19-21.7-13.79l-0.51-.35c0.52-.39,1-0.76,1.47-1.1a20.09,20.09,0,0,1,8.16-3.67,8,8,0,0,1,5.73,1q11.37,7.07,22.75,14.11c1.71,1.06,3.43,2.09,5.15,3.13,0.17,0.1.34,0.18,0.65,0.34-0.22-1-.4-1.91-0.6-2.79-2.18-9.65-4.2-19.34-6.63-28.93-1.22-4.8.48-8.32,3.59-11.54,1.22-1.26,2.65-2.32,4-3.47,0.13-.11.25-0.24,0.37-0.36h0.13Z",
          "transform": "translate(-239 -257.91)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M373,317.27c-0.28.77-.55,1.54-0.83,2.31a21.18,21.18,0,0,1-3.68,6.56,8.7,8.7,0,0,1-6.71,3.2c-2.87.07-5.74,0.38-8.61,0.6s-5.78.46-8.67,0.71l-8.46.75-5.13.45-1.39.15c1.16,1,2.2,2,3.24,2.9l20.64,18.2a5.4,5.4,0,0,0,.51.43c2.73,1.86,3.8,4.58,3.91,7.75a25,25,0,0,1-1.51,8.92,1.54,1.54,0,0,1-.16.26c-1.52-1.29-3-2.55-4.5-3.83Q338.23,355,324.83,343.29a11.74,11.74,0,0,1-4.43-9.7c0.22-4.15,1.54-7.92,4.79-10.75a11.7,11.7,0,0,1,6.78-2.73q5.09-.49,10.19-0.9c3.27-.26,6.54-0.48,9.81-0.71,3.63-.26,7.25-0.52,10.88-0.76,3.38-.22,6.77-0.41,10.15-0.61v0.13Z",
          "transform": "translate(-239 -257.91)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "cls-1",
          "d": "M319.17,304.74c1-.56,1.84-1,2.69-1.53l24.94-14.47a12.09,12.09,0,0,1,3.89-1.88,10.62,10.62,0,0,1,5,.35,24.63,24.63,0,0,1,8.25,4.58c-0.17.13-.29,0.24-0.43,0.32l-35,20.69a13.28,13.28,0,0,1-6.51,2,12.27,12.27,0,0,1-7.49-2.45,13.3,13.3,0,0,1-5.71-7.58,13.17,13.17,0,0,1,.1-6.68q5.12-19.65,10.28-39.28c0-.17.11-0.33,0.21-0.65,1.12,0.94,2.21,1.78,3.22,2.71a17.09,17.09,0,0,1,4.33,5.81,8.57,8.57,0,0,1,.37,5.74c-2.11,8.05-4.14,16.13-6.19,24.2-0.62,2.46-1.21,4.93-1.81,7.39C319.26,304.19,319.24,304.39,319.17,304.74Z",
          "transform": "translate(-239 -257.91)",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
    ],
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <defs>
            <style>
              .cls-1{fill:#fff;}
            </style>
          </defs>,
          <title>
            istelle-white-icon
          </title>,
          <path
            className="cls-1"
            d="M239,317.54c4.92,0.32,9.84.63,14.76,1q8.71,0.62,17.42,1.3c3.31,0.26,6.63.47,9.93,0.88,5.52,0.68,9,3.85,10.66,9.11a12.42,12.42,0,0,1-3.62,14.09q-11.6,10-23.24,19.92c-2.63,2.25-5.3,4.45-8,6.67l-0.71.57c-0.37-1.37-.76-2.64-1.05-3.93a17.53,17.53,0,0,1-.21-7.82,7.82,7.82,0,0,1,2.53-4.31l23.24-20.12c0.79-.68,1.57-1.38,2.52-2.22-0.88-.1-1.57-0.18-2.26-0.24-4.42-.4-8.84-0.8-13.26-1.18q-6-.51-12-1c-2.11-.16-4.23-0.26-6.34-0.42A8.2,8.2,0,0,1,243.9,327a22.4,22.4,0,0,1-4.62-8.52c-0.07-.22-0.19-0.44-0.28-0.65v-0.27Z"
            transform="translate(-239 -257.91)"
          />,
          <path
            className="cls-1"
            d="M278.66,387.89c1.88-4.77,3.73-9.55,5.65-14.3q4.93-12.21,9.93-24.4a12.23,12.23,0,0,1,6.07-6.77,14.17,14.17,0,0,1,7-1.19,12.53,12.53,0,0,1,6.32,1.92,12.08,12.08,0,0,1,4.59,5.62q4.89,11.83,9.67,23.7c2.07,5.12,4.05,10.28,6.07,15.42h-2.55c-1.52-.23-3.05-0.37-4.54-0.72-3.76-.87-6.53-2.74-8.05-6.62-3.93-10-8.14-19.87-12.25-29.79a5.1,5.1,0,0,0-.28-0.49c-0.09.17-.15,0.27-0.19,0.37q-6.38,15.48-12.75,31a7.84,7.84,0,0,1-2.55,3.37,13.48,13.48,0,0,1-5.63,2.36c-1.15.23-2.32,0.38-3.48,0.56h-3.08Z"
            transform="translate(-239 -257.91)"
          />,
          <path
            className="cls-1"
            d="M291.26,257.91c1,4,1.94,7.92,2.87,11.89,2.25,9.66,4.51,19.31,6.7,29,1.28,5.67-.77,10.42-5.79,13.85a11.89,11.89,0,0,1-13.46.62c-4.3-2.63-8.62-5.24-12.88-7.92-7.25-4.57-14.47-9.19-21.7-13.79l-0.51-.35c0.52-.39,1-0.76,1.47-1.1a20.09,20.09,0,0,1,8.16-3.67,8,8,0,0,1,5.73,1q11.37,7.07,22.75,14.11c1.71,1.06,3.43,2.09,5.15,3.13,0.17,0.1.34,0.18,0.65,0.34-0.22-1-.4-1.91-0.6-2.79-2.18-9.65-4.2-19.34-6.63-28.93-1.22-4.8.48-8.32,3.59-11.54,1.22-1.26,2.65-2.32,4-3.47,0.13-.11.25-0.24,0.37-0.36h0.13Z"
            transform="translate(-239 -257.91)"
          />,
          <path
            className="cls-1"
            d="M373,317.27c-0.28.77-.55,1.54-0.83,2.31a21.18,21.18,0,0,1-3.68,6.56,8.7,8.7,0,0,1-6.71,3.2c-2.87.07-5.74,0.38-8.61,0.6s-5.78.46-8.67,0.71l-8.46.75-5.13.45-1.39.15c1.16,1,2.2,2,3.24,2.9l20.64,18.2a5.4,5.4,0,0,0,.51.43c2.73,1.86,3.8,4.58,3.91,7.75a25,25,0,0,1-1.51,8.92,1.54,1.54,0,0,1-.16.26c-1.52-1.29-3-2.55-4.5-3.83Q338.23,355,324.83,343.29a11.74,11.74,0,0,1-4.43-9.7c0.22-4.15,1.54-7.92,4.79-10.75a11.7,11.7,0,0,1,6.78-2.73q5.09-.49,10.19-0.9c3.27-.26,6.54-0.48,9.81-0.71,3.63-.26,7.25-0.52,10.88-0.76,3.38-.22,6.77-0.41,10.15-0.61v0.13Z"
            transform="translate(-239 -257.91)"
          />,
          <path
            className="cls-1"
            d="M319.17,304.74c1-.56,1.84-1,2.69-1.53l24.94-14.47a12.09,12.09,0,0,1,3.89-1.88,10.62,10.62,0,0,1,5,.35,24.63,24.63,0,0,1,8.25,4.58c-0.17.13-.29,0.24-0.43,0.32l-35,20.69a13.28,13.28,0,0,1-6.51,2,12.27,12.27,0,0,1-7.49-2.45,13.3,13.3,0,0,1-5.71-7.58,13.17,13.17,0,0,1,.1-6.68q5.12-19.65,10.28-39.28c0-.17.11-0.33,0.21-0.65,1.12,0.94,2.21,1.78,3.22,2.71a17.09,17.09,0,0,1,4.33,5.81,8.57,8.57,0,0,1,.37,5.74c-2.11,8.05-4.14,16.13-6.19,24.2-0.62,2.46-1.21,4.93-1.81,7.39C319.26,304.19,319.24,304.39,319.17,304.74Z"
            transform="translate(-239 -257.91)"
          />,
        ],
        "data-name": "Layer 1",
        "id": "Layer_1",
        "viewBox": "0 0 134 129.98",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <style>
              .cls-1{fill:#fff;}
            </style>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": ".cls-1{fill:#fff;}",
            },
            "ref": null,
            "rendered": ".cls-1{fill:#fff;}",
            "type": "style",
          },
          "type": "defs",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "istelle-white-icon",
          },
          "ref": null,
          "rendered": "istelle-white-icon",
          "type": "title",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M239,317.54c4.92,0.32,9.84.63,14.76,1q8.71,0.62,17.42,1.3c3.31,0.26,6.63.47,9.93,0.88,5.52,0.68,9,3.85,10.66,9.11a12.42,12.42,0,0,1-3.62,14.09q-11.6,10-23.24,19.92c-2.63,2.25-5.3,4.45-8,6.67l-0.71.57c-0.37-1.37-.76-2.64-1.05-3.93a17.53,17.53,0,0,1-.21-7.82,7.82,7.82,0,0,1,2.53-4.31l23.24-20.12c0.79-.68,1.57-1.38,2.52-2.22-0.88-.1-1.57-0.18-2.26-0.24-4.42-.4-8.84-0.8-13.26-1.18q-6-.51-12-1c-2.11-.16-4.23-0.26-6.34-0.42A8.2,8.2,0,0,1,243.9,327a22.4,22.4,0,0,1-4.62-8.52c-0.07-.22-0.19-0.44-0.28-0.65v-0.27Z",
            "transform": "translate(-239 -257.91)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M278.66,387.89c1.88-4.77,3.73-9.55,5.65-14.3q4.93-12.21,9.93-24.4a12.23,12.23,0,0,1,6.07-6.77,14.17,14.17,0,0,1,7-1.19,12.53,12.53,0,0,1,6.32,1.92,12.08,12.08,0,0,1,4.59,5.62q4.89,11.83,9.67,23.7c2.07,5.12,4.05,10.28,6.07,15.42h-2.55c-1.52-.23-3.05-0.37-4.54-0.72-3.76-.87-6.53-2.74-8.05-6.62-3.93-10-8.14-19.87-12.25-29.79a5.1,5.1,0,0,0-.28-0.49c-0.09.17-.15,0.27-0.19,0.37q-6.38,15.48-12.75,31a7.84,7.84,0,0,1-2.55,3.37,13.48,13.48,0,0,1-5.63,2.36c-1.15.23-2.32,0.38-3.48,0.56h-3.08Z",
            "transform": "translate(-239 -257.91)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M291.26,257.91c1,4,1.94,7.92,2.87,11.89,2.25,9.66,4.51,19.31,6.7,29,1.28,5.67-.77,10.42-5.79,13.85a11.89,11.89,0,0,1-13.46.62c-4.3-2.63-8.62-5.24-12.88-7.92-7.25-4.57-14.47-9.19-21.7-13.79l-0.51-.35c0.52-.39,1-0.76,1.47-1.1a20.09,20.09,0,0,1,8.16-3.67,8,8,0,0,1,5.73,1q11.37,7.07,22.75,14.11c1.71,1.06,3.43,2.09,5.15,3.13,0.17,0.1.34,0.18,0.65,0.34-0.22-1-.4-1.91-0.6-2.79-2.18-9.65-4.2-19.34-6.63-28.93-1.22-4.8.48-8.32,3.59-11.54,1.22-1.26,2.65-2.32,4-3.47,0.13-.11.25-0.24,0.37-0.36h0.13Z",
            "transform": "translate(-239 -257.91)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M373,317.27c-0.28.77-.55,1.54-0.83,2.31a21.18,21.18,0,0,1-3.68,6.56,8.7,8.7,0,0,1-6.71,3.2c-2.87.07-5.74,0.38-8.61,0.6s-5.78.46-8.67,0.71l-8.46.75-5.13.45-1.39.15c1.16,1,2.2,2,3.24,2.9l20.64,18.2a5.4,5.4,0,0,0,.51.43c2.73,1.86,3.8,4.58,3.91,7.75a25,25,0,0,1-1.51,8.92,1.54,1.54,0,0,1-.16.26c-1.52-1.29-3-2.55-4.5-3.83Q338.23,355,324.83,343.29a11.74,11.74,0,0,1-4.43-9.7c0.22-4.15,1.54-7.92,4.79-10.75a11.7,11.7,0,0,1,6.78-2.73q5.09-.49,10.19-0.9c3.27-.26,6.54-0.48,9.81-0.71,3.63-.26,7.25-0.52,10.88-0.76,3.38-.22,6.77-0.41,10.15-0.61v0.13Z",
            "transform": "translate(-239 -257.91)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "cls-1",
            "d": "M319.17,304.74c1-.56,1.84-1,2.69-1.53l24.94-14.47a12.09,12.09,0,0,1,3.89-1.88,10.62,10.62,0,0,1,5,.35,24.63,24.63,0,0,1,8.25,4.58c-0.17.13-.29,0.24-0.43,0.32l-35,20.69a13.28,13.28,0,0,1-6.51,2,12.27,12.27,0,0,1-7.49-2.45,13.3,13.3,0,0,1-5.71-7.58,13.17,13.17,0,0,1,.1-6.68q5.12-19.65,10.28-39.28c0-.17.11-0.33,0.21-0.65,1.12,0.94,2.21,1.78,3.22,2.71a17.09,17.09,0,0,1,4.33,5.81,8.57,8.57,0,0,1,.37,5.74c-2.11,8.05-4.14,16.13-6.19,24.2-0.62,2.46-1.21,4.93-1.81,7.39C319.26,304.19,319.24,304.39,319.17,304.74Z",
            "transform": "translate(-239 -257.91)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
      ],
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
